extensions/AbuseFilter/Views/AbuseFilterViewDiff.php

```diff
@@ -135,7 +135,7 @@ function loadFromHistoryRow( $row ) {
 			'meta' => array(
 				'history_id' => $row->afh_id,
 				'modified_by' => $row->afh_user,
-				'modified_by_text' => $row->afh_user_text,
+				'modified_by_text' => User::getUsername( $row->afh_user, $row->afh_user_text ),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "modified_by_text" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)